packages/angular/src/schematics/stories/stories.ts

```diff
@@ -10,13 +10,15 @@ import {
 import { SyntaxKind } from 'typescript';
 import { getDecoratorMetadata, getTsSourceFile } from '../../utils/ast-utils';
 import { projectRootPath } from '@nrwl/workspace/src/utils/project-type';
-import { findNodes } from '@nrwl/workspace/src/utils/ast-utils';
+import {
+  findNodes,
+  getProjectConfig,
+} from '@nrwl/workspace/src/utils/ast-utils';
```

Comment:
sorry didn't catch these the first round. Can we import them from `@nrwl/workspace` directly rather than having the deep import? I think they're both exposed

Reply:
Ohh sure!